### DISCUSSION TITLE
setting the logging level

### DISCUSSION THREAD

**Discussion by zytegalaxy** [ubopod/ubo_app#7] [Ideas]
In `ubo.logging.py` the logging level if hard coded. For release build, we need to set a different level of debugging. 

 stdout_handler.setLevel(logging.DEBUG)

Probably need to define a way to set this.

**Comment by sassanh** [comment]
We already have an environment variable for this purpose, it is `UBO_LOG_LEVEL`
I reduced the default verboseness of the application, but one has the option to increase it using `UBO_LOG_LEVEL`.
Is it enough or do we need more?
